doc/selectors.rst

```diff
@@ -64,17 +66,19 @@ of the face.
 
 The axis used in the listing below are for illustration: any axis would work similarly in each case.
 
-=========   ======================================  =======================================================  ==========================
-Selector    Selects                                 Selector Class                                           # objects returned
-=========   ======================================  =======================================================  ==========================
-+Z          Faces with normal in +z direction       :py:class:`cadquery.DirectionSelector`                   0 or 1
-\|Z         Faces parallel to xy plane              :py:class:`cadquery.ParallelDirSelector`                 0..many
--X          Faces with  normal in neg x direction   :py:class:`cadquery.DirectionSelector`                   0..many
-#Z          Faces perpendicular to z direction      :py:class:`cadquery.PerpendicularDirSelector`            0..many
-%Plane      Faces of type plane                     :py:class:`cadquery.TypeSelector`                        0..many
->Y          Face farthest in the positive y dir     :py:class:`cadquery.DirectionMinMaxSelector`             0 or 1
-<Y          Face farthest in the negative y dir     :py:class:`cadquery.DirectionMinMaxSelector`             0 or 1
-=========   ======================================  =======================================================  ==========================
+=========   =======================================  =======================================================  ==========================
+Selector    Selects                                  Selector Class                                           # objects returned
+=========   =======================================  =======================================================  ==========================
++Z          Faces with normal in +z direction        :py:class:`cadquery.DirectionSelector`                   0..many
+\|Z         Faces parallel to xy plane               :py:class:`cadquery.ParallelDirSelector`                 0..many
+-X          Faces with  normal in neg x direction    :py:class:`cadquery.DirectionSelector`                   0..many
+#Z          Faces perpendicular to z direction       :py:class:`cadquery.PerpendicularDirSelector`            0..many
+%Plane      Faces of type plane                      :py:class:`cadquery.TypeSelector`                        0..many
+>Y          Face farthest in the positive y dir      :py:class:`cadquery.DirectionMinMaxSelector`             0..many
+<Y          Face farthest in the negative y dir      :py:class:`cadquery.DirectionMinMaxSelector`             0..many
+>Y[-2]      2nd Face farthest in the positive y dir  :py:class:`cadquery.DirectionMinMaxSelector`             0..many
```

Comment:
The index on this would be -2 instead of 2? I had assumed that since you're selecting the farthest in the direction, the index would always be positive unless you're wrapping around to the beginning of the collection.